Entity/Components/StatComponent.gd

```diff
@@ -6,12 +6,12 @@ class_name StatComponent
 ## have different stats based on the Entity type.
 
 
-var _stats: EntityStats = EntityStats.new()
-
-
-func set_stats(new_stats: EntityStats) -> void:
-	_stats = new_stats
+var stats: EntityStats = EntityStats.new():
+	set(value):
+		stats = value
+	get:
+		return stats
 
 
 func get_stats() -> EntityStats:
```

Comment:
this is not needed anymore since we have the getter for stats already
if you call `get_stats` you basically cal `return return stats`

Reply:
So if I remove `get_stats` I'll have to update all of our calls to `get_stats` to just `stats` right? Is that okay?

Reply:
seems okay yes :thinking: 
you could probably do a search in all files and replace, then reload and check there are no problems
I don't think there are other functions called `get_stats`